pkg/splunk/common/urls_testingcode.go

```diff
@@ -0,0 +1,409 @@
+package common
+
+// Endpoints
+// List of all endpoints (Full URls) used by the Testing code
+
+const (
+
+	//*****************
+	// URLs based on localhost
+	//*****************
+
+	//TestCommonOutput = "?count=0&output_mode=json"
+	TestCommonOutput = QM + CountZero + Amp + JSONOutput
+
+	//PeerInfoJSON = "https://localhost:8089/services/cluster/slave/info?count=0&output_mode=json"
+	PeerInfoJSON = LocalURL + PeerInfoAPI + TestCommonOutput
+
+	//PeerDecommission = "https://localhost:8089/services/cluster/slave/control/control/decommission?
+	PeerDecommission = LocalURL + DecommissionAPI
+
+	//ApplyBundleTest = "https://localhost:8089/services/cluster/master/control/default/apply"
+	ApplyBundleTest = LocalURL + ApplyBundleAPI
+
+	//ManagerInfoTest = "https://localhost:8089/services/cluster/master/info?count=0&output_mode=json"
+	ManagerInfoTest = LocalURL + ManagerInfoAPI + TestCommonOutput
+
+	//ManagerInfoTestJSON = "https://localhost:8089/services/cluster/master/info?output_mode=json"
+	ManagerInfoTestJSON = LocalURL + ManagerInfoAPI + JSONOutput
```

Comment:
Considering my other comments, assuming if we are going with this kind of tokenization, wondering if a naming convention like `UriServiersClusterMasterInfo`, `UriServicesClusterMasterPeers`, `UriServicesClusterMasterPeersCount` etc..  which gives us better readability wherever these are being used.

Reply:
Sounds like a good idea to use RHS naming convention. Will change if we keep this approach.

Reply:
Changed the naming convention in the latest commit. Will complete the changes after our discussion later today.

Reply:
Resolved in the latest commit which we reduced the amount tokenizations and improved the naming convention.